ceph/ceph_admin/daemon.py

```diff
@@ -0,0 +1,42 @@
+"""Cephadm orchestration host operations."""
+import logging
+
+from .add import AddMixin
+from .common import config_dict_to_string
+from .orch import Orch
+
+logger = logging.getLogger(__name__)
+
+
+class Daemon(AddMixin, Orch):
```

Comment:
I Agree with only AddMixin now for OSD creation, rest we can take up later with available support.